ingestors/maven_central.go

```diff
@@ -1,12 +1,14 @@
 package ingestors
 
 import (
+	"time"
+
 	"github.com/librariesio/depper/data"
 	log "github.com/sirupsen/logrus"
-	"time"
 )
 
 const mavenCentralSchedule = "@every 12h"
+const ttlHours = 168 * time.Hour // 1 Week
```

Comment:
Could probably just call it `ttl` since it's a Duration and doesn't store the context of hours.

Reply:
Totally. I started with an int.